### ISSUE TITLE
[Consistency] User encounters a component or pattern that isn't considered a best practice.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#78125]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** OCTO Health Apps
**Product:** MHV Medical Records
**Feature:** After-Visit Summary

## Findings details
**VA.gov Experience Standard - issue:** User encounters a component or pattern that isn't considered a best practice.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Collab Cycle Reviewer:** @rferguson13 (Content), @briandeconinck (Accessibility)

### Description

The phone numbers don't use the `va-telephone` component. 

## Recommended action
Please revise so all phone numbers listed use that component

## References

- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Telephone ](https://design.va.gov/components/telephone)
- **Design System Pattern:** N/A
- **Content Style Guide:** [Links](https://design.va.gov/content-style-guide/links)
- **Context:** N/A
- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
